examples/chefapi_examples/recipes/default.rb

```diff
@@ -4,4 +4,7 @@
 #
 # Copyright:: 2019, The Authors, All Rights Reserved.
 
+hostname 'testhost'
```

Comment:
Moving the hostname switching here ensured that the auto-generated self-signed certificate had a known name. Using the `hostname` resource also removes the need to manually tweak the hosts file.